infra/tpu-pytorch-releases/artifacts.auto.tfvars

```diff
@@ -23,6 +23,60 @@ nightly_builds = [
 
 # Built on push to specific tag.
 versioned_builds = [
+  # Remove libtpu from PyPI builds
+  {
+    git_tag         = "v2.3.0-rc1"
+    package_version = "2.3.0-rc1"
+    pytorch_git_rev = "v2.3.0-rc1"
+    accelerator     = "tpu"
+    bundle_libtpu   = "0"
+  },
+  {
+    git_tag         = "v2.3.0-rc1"
+    package_version = "2.3.0-rc1"
+    pytorch_git_rev = "v2.3.0-rc1"
+    accelerator     = "tpu"
+    python_version  = "3.9"
+    bundle_libtpu   = "0"
+  },
+  {
+    git_tag         = "v2.3.0-rc1"
+    package_version = "2.3.0-rc1"
+    pytorch_git_rev = "v2.3.0-rc1"
+    accelerator     = "tpu"
+    python_version  = "3.10"
+    bundle_libtpu   = "0"
+  },
+  {
+    git_tag         = "v2.3.0-rc1"
+    package_version = "2.3.0-rc1"
+    pytorch_git_rev = "v2.3.0-rc1"
+    accelerator     = "tpu"
+    python_version  = "3.11"
+    bundle_libtpu   = "0"
+  },
+  # Bundle libtpu for Kaggle
+  {
+    git_tag         = "v2.3.0-rc1"
+    package_version = "2.3.0-rc1+libtpu"
+    pytorch_git_rev = "v2.3.0-rc1"
+    accelerator     = "tpu"
+    python_version  = "3.10"
+    bundle_libtpu   = "1"
+  },
+  {
+    git_tag         = "v2.3.0-rc1"
+    package_version = "2.3.0-rc1"
```

Comment:
The CUDA targets also need a `pytorch_git_rev`

Reply:
Thanks, let me add.